src/Mutator/Unwrap/UnwrapArrayUintersectAssoc.php

```diff
@@ -76,7 +76,16 @@ public static function getDefinition(): ?Definition
 TXT
             ,
             MutatorCategory::SEMANTIC_REDUCTION,
-            null
+            null,
+            <<<'DIFF'
+- $x = array_uintersect_assoc(
+-     ['foo' => 'bar'],
+-     ['baz' => 'bar'],
+-     $value_compare_func
+- );
++ $x = ['foo' => 'bar'];
```

Comment:
is that correct? because this could be simplified to `$x = ['baz' => 'bar'];` right? Or we still do this in case there is a call with side-effects?

Reply:
Yes, this is how it works - producing N-1 mutatations where N is the number of arguments:

https://github.com/infection/infection/blob/1f0569f37432228f0ccb1d326d53194d06bb728f/tests/phpunit/Mutator/Unwrap/UnwrapArrayUintersectAssocTest.php#L58-L74

Reply:
Would something like that still work?

```
            <<<'DIFF'
- $x = array_uintersect_assoc(
-     ['foo' => 'bar'],
-     ['baz' => 'bar'],
-     $value_compare_func
- );
# Mutation °1
+ $x = ['foo' => 'bar'];
# Mutation °2
+ $x = ['baz' => 'bar'];
```

Reply:
do you mean we need to point the number of mutation in this diffs? If so - makes sense.. can do in another PR

Reply:
Yes I find it a bit confusing otherwise as it's not clear that it's actually two mutations not one. It is a suggestion however, maybe there is a better way to convey it

Reply:
agree, will do, thanks